assets/js/modules/analytics/common/adsense-link-cta.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Analytics AdSense Link CTA component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import CTA from '../../../components/notifications/cta';
+
+export default function AdSenseLinkCTA() {
+	return (
+		<CTA
+			title={ __( 'Link Analytics and AdSense', 'google-site-kit' ) }
+			description={ __( 'You need to link Analytics and AdSense to get reports for your top earning pages.', 'google-site-kit' ) }
```

Comment:
This sounds a bit like an instruction rather than a choice the user has, if they want. 
Perhaps something like "Get reports for your top earning pages by linking your Analytics and AdSense accounts."?

Reply:
Yeah, agree with Evan on the wording -- let's make clear what benefit they're getting from this if they decide to go through with the action. 

Other than that, why are the blue rectangles different sizes? The one in the topmost section has a white margin, while the other two span the entire widget. (Not blocking, just looks strange, would be good to have uniform UI.)

Reply:
```suggestion
			description={ __( 'Get reports for your top earning pages by linking your Analytics and AdSense accounts.', 'google-site-kit' ) }
```